### PR TITLE
[hotfix] Fixed typo in docs

### PR DESCRIPTION
**Pull request by elbaulp**

### CODE REVIEW

File: docs/monitoring/metrics.md [hunk 1]
```diff
@@ -426,7 +426,7 @@ class MyMapper extends RichMapFunction[Long,Long] {
 
 Every metric is assigned an identifier and a set of key-value pairs under which the metric will be reported.
 
-THe identifier is based on 3 components: the user-defined name when registering the metric, an optional user-defined scope and a system-provided scope.
+The identifier is based on 3 components: the user-defined name when registering the metric, an optional user-defined scope and a system-provided scope.
```
Comment (yanghua) [inline docs/monitoring/metrics.md]: @elbaulp I think use "a user-defined name" to replace "the user-defined name" would be better, what's your opinion? @tillrohrmann

Comment (tillrohrmann) [reply docs/monitoring/metrics.md]: I think you are right @yanghua. Will change it will merging this PR.

**Comment by tillrohrmann** [comment]
Thanks for fixing the typo @elbaulp. Merging this PR.

**Comment by elbaulp** [comment]
@tillrohrmann You're welcome :-)

**Pull request closed without merging**